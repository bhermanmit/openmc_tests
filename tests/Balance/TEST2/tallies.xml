<?xml version="1.0"?>
<tallies>

  <tally id="1">
    <filters>
      <mesh>1</mesh>
    </filters>
    <macros>flux total scatter-1</macros>
  </tally>

  <tally id="2">
    <filters>
      <mesh>1</mesh>
    </filters>
    <macros>nu-scatter nu-fission</macros>
  </tally>

  <tally id="3">
    <filters>
      <mesh>1</mesh>
    </filters>
    <macros>current</macros>
  </tally>

  <mesh id="1">
    <type> rectangular </type>
    <origin> -25 -10 -10 </origin>
    <dimension> 1 1 1 </dimension>
    <width> 50.0 20.0 20.0 </width>

  </mesh>


</tallies>
