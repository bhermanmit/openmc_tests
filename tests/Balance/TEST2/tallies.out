
 ============================>     TALLY 1     <============================

 Mesh Index (1, 1, 1)
   Flux                        36.2998        +/- 6.34920E-03
   Total Reaction Rate         3.38122        +/- 6.52082E-04
   First Scattering Moment     0.962257       +/- 2.69251E-04

 ============================>     TALLY 2     <============================

 Mesh Index (1, 1, 1)
   Scattering Production Rate  2.73012        +/- 6.44516E-04
   Nu-Fission Rate             1.76634        +/- 2.99374E-04

 ============================>     TALLY 3     <============================

 Mesh Index (1, 1, 1)
     Outgoing Current to Left     0.180355       +/- 5.95843E-03
     Incoming Current from Left   0.0            +/- 0.0
     Incoming Current from Right  0.0            +/- 0.0
     Outgoing Current to Right    0.180303       +/- 6.00120E-03
     Outgoing Current to Back     0.0            +/- 0.0
     Incoming Current from Back   0.0            +/- 0.0
     Incoming Current from Front  0.0            +/- 0.0
     Outgoing Current to Front    0.0            +/- 0.0
     Outgoing Current to Bottom   0.0            +/- 0.0
     Incoming Current from Bottom 0.0            +/- 0.0
     Incoming Current from Top    0.0            +/- 0.0
     Outgoing Current to Top      0.0            +/- 0.0
